Dockerfile

```diff
@@ -0,0 +1,132 @@
+# (https://github.com/docker-library/openjdk/pull/322#discussion_r286839190)
+  gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys EAC843EBD3EFDB98CC772FADA5CD6035332FA671; \
+  gpg --batch --list-sigs --keyid-format 0xLONG CA5F11C6CE22644D42C6AC4492EF8D39DC13168F \
+    | tee /dev/stderr \
+    | grep '0xA5CD6035332FA671' \
+    | grep 'Andrew Haley'; \
+  gpg --batch --verify openjdk.tgz.asc openjdk.tgz; \
+  gpgconf --kill all; \
+  rm -rf "$GNUPGHOME"; \
+  \
+  mkdir -p "$JAVA_HOME"; \
+    tar --extract \
+    --file openjdk.tgz \
+    --directory "$JAVA_HOME" \
+    --strip-components 1 \
+    --no-same-owner \
+  ; \
+  rm openjdk.tgz*; \
+  \
+# TODO strip "demo" and "man" folders?
+  \
+# update "cacerts" bundle to use Debian's CA certificates (and make sure it stays up-to-date with changes to Debian's store)
+# see https://github.com/docker-library/openjdk/issues/327
+# http://rabexc.org/posts/certificates-not-working-java#comment-4099504075
+# https://salsa.debian.org/java-team/ca-certificates-java/blob/3e51a84e9104823319abeb31f880580e46f45a98/debian/jks-keystore.hook.in
+# https://git.alpinelinux.org/aports/tree/community/java-cacerts/APKBUILD?id=761af65f38b4570093461e6546dcf6b179d2b624#n29
+  { \
+  echo '#!/usr/bin/env bash'; \
+  echo 'set -Eeuo pipefail'; \
+  echo 'if ! [ -d "$JAVA_HOME" ]; then echo >&2 "error: missing JAVA_HOME environment variable"; exit 1; fi'; \
+# 8-jdk uses "$JAVA_HOME/jre/lib/security/cacerts" and 8-jre and 11+ uses "$JAVA_HOME/lib/security/cacerts" directly (no "jre" directory)
+  echo 'cacertsFile=; for f in "$JAVA_HOME/lib/security/cacerts" "$JAVA_HOME/jre/lib/security/cacerts"; do if [ -e "$f" ]; then cacertsFile="$f"; break; fi; done'; \
+  echo 'if [ -z "$cacertsFile" ] || ! [ -f "$cacertsFile" ]; then echo >&2 "error: failed to find cacerts file in $JAVA_HOME"; exit 1; fi'; \
+  echo 'trust extract --overwrite --format=java-cacerts --filter=ca-anchors --purpose=server-auth "$cacertsFile"'; \
+} > /etc/ca-certificates/update.d/docker-openjdk; \
+  chmod +x /etc/ca-certificates/update.d/docker-openjdk; \
+  /etc/ca-certificates/update.d/docker-openjdk; \
+  \
+# https://github.com/docker-library/openjdk/issues/331#issuecomment-498834472
+  find "$JAVA_HOME/lib" -name '*.so' -exec dirname '{}' ';' | sort -u > /etc/ld.so.conf.d/docker-openjdk.conf; \
+  ldconfig; \
+  \
+# basic smoke test
+  javac --version; \
+  java --version
+
+## ----------------------------------------------------------------------------
+## --- JAVA INSTALLATION FINISHED ---------------------------------------------
+## ----------------------------------------------------------------------------
+
+## ----------------------------------------------------------------------------
+## --- INSTALLING SUITECLOUD (Nodejs CLI) -------------------------------------
+## ----------------------------------------------------------------------------
```

Comment:
The real work starts here. We add suitecloud and sdfcli to the docker image, since at this point it has Node12.14 and java 11

Reply:
Why do we install sdfcli?

Reply:
I've added it because it's a requirement of the user story PDPDEVTOOL-2660

Reply:
@margabit We had a discussion about whether to install both CLIs and dicided to have two in one container. At this point Java CLI provides a bit more commands so maybe it's good to have both of them in the container for CI. I think we may remove sdfcli later. Do you have concerns about having both installed?

Reply:
Ok, sounds good!